packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -262,6 +262,7 @@ const Breakouts = WebexPlugin.extend({
     this.set(params);
     this.set('groups', params.groups);
     this.set('startTime', params.startTime);
+    this.set('status', params.status);
```

Comment:
question: Why is this (and the groups, startTime) needed? Line 262 sets all params to the breakouts, manually setting part of params to something with the same name shouldn't be necessary

Reply:
I've worked this out myself. This is so that status, groups, startTime will be overwritten when they aren't provided in the params object. This isn't very obvious, maybe work a comment here.